docs/conceptual/list.chunkbysize['t]-function-[fsharp].md

```diff
@@ -0,0 +1,80 @@
+---
+title: List.chunkBySize<'T> Function (F#)
+description: List.chunkBySize<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: banashek
+---
+
+# List.chunkBySize<'T> Function (F#)
+
+Divides the input list into chunks of size at most chunkSize.
+
+**Namespace/Module Path:** Microsoft.FSharp.Collections.List
+
+**Assembly:** FSharp.Core (in FSharp.Core.dll)
+
+
+## Syntax
+
+```fsharp
+// Signature:
+List.chunkBySize : int -> 'T list -> 'T list list
```

Comment:
This makes sense. I was following other documentation, but I agree that signatures should be copy-pasted from the .fsi files.